src/Backend/Modules/Location/Domain/Location.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+namespace Backend\Modules\Location\Domain;
+
+use Common\Locale;
+use Backend\Core\Language\Locale as BackendLocale;
+use Frontend\Core\Language\Locale as FrontendLocale;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="location")
```

Comment:
Moet die table-name gespecificeerd worden, of doe je het voor backwords-compatible te zijn?

Reply:
It's to be backwards compatible but it's not completely necessary (there are database changes anyway) so I might remove these

Reply:
see other comment